imageregistry/v1/stable.config.testsuite.yaml

```diff
@@ -16,3 +16,101 @@ tests:
         logLevel: Normal
         operatorLogLevel: Normal
         replicas: 1
+  - name: Should be able to configure internal storage network access for Azure
+    initial: |
+      apiVersion: imageregistry.operator.openshift.io/v1
+      kind: Config
+      spec:
+        replicas: 1
+        storage:
+          azure:
+            networkAccess:
+              type: Internal
+              internal:
+                vnetName: foo-bar
+                subnetName: bar_foo
+                privateEndpointName: my-private-endpoint
+    expected: |
+      apiVersion: imageregistry.operator.openshift.io/v1
+      kind: Config
+      spec:
+        logLevel: Normal
+        operatorLogLevel: Normal
+        replicas: 1
+        storage:
+          azure:
+            networkAccess:
+              type: Internal
+              internal:
+                vnetName: foo-bar
+                subnetName: bar_foo
+                privateEndpointName: my-private-endpoint
+  - name: Should be able to configure internal storage network access for Azure without details
+    initial: |
+      apiVersion: imageregistry.operator.openshift.io/v1
+      kind: Config
+      spec:
+        replicas: 1
+        storage:
+          azure:
+            networkAccess:
+              type: Internal
+    expected: |
+      apiVersion: imageregistry.operator.openshift.io/v1
+      kind: Config
+      spec:
+        logLevel: Normal
+        operatorLogLevel: Normal
+        replicas: 1
+        storage:
+          azure:
+            networkAccess:
+              type: Internal
+  - name: Should reject configuration with invalid vnet name in internal storage network access config
+    initial: |
+      apiVersion: imageregistry.operator.openshift.io/v1
+      kind: Config
+      spec:
+        replicas: 1
+        storage:
+          azure:
+            networkAccess:
+              type: Internal
+              internal:
+                vnetName: .invalid
+    expected: |
```

Comment:
`expected` can be removed when `expectedError` is set

Reply:
done